sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -451,7 +404,12 @@ public long count(VisitCriteria visitCriteria) {
 			visitRoot.get(Visit.REPORT_LAT),
 			visitRoot.get(Visit.REPORT_LON),
 			visitRoot.get(Visit.ORIGIN),
-			personJoin.get(Person.UUID));
+			personJoin.get(Person.UUID),
+			caseService.jurisdictionSelector(
```

Comment:
would be handful to create a method for this so it would not be duplicated and it will be easier to implement the visits for event participants for example if needed in the future